lib/src/robot_interface/robot_interface.cpp

```diff
@@ -60,15 +60,24 @@ RobotInterface::RobotInterface(string name, string limb, bool no_robot, bool use
     _filt_force.push_back(0.0);
     _filt_force.push_back(0.0);
 
+    _filt_updated = 0;
+    _filt_change.push_back(0.0);
+    _filt_change.push_back(0.0);
+    _filt_change.push_back(0.0);
+
     if (getLimb()=="left")
     {
         _n.param<double>("force_threshold_left",  force_thres, FORCE_THRES_L);
+        _n.param<double>("force_filt_variance_left", filt_variance, FORCE_FILT_VAR_L);
+        _n.param<double>("rel_force_threshold_left", rel_force_thres, REL_FORCE_THRES_L);
     }
     else if (getLimb()=="right")
     {
         _n.param<double>("force_threshold_right", force_thres, FORCE_THRES_R);
     }
     ROS_INFO("[%s] Force Threshold : %g", getLimb().c_str(), force_thres);
+    ROS_INFO("[%s] Force Filter Variance: %g", getLimb().c_str(), filt_variance);
+    ROS_INFO("[%s] Relative Force Threshold: %g", getLimb().c_str(), rel_force_thres);
```

Comment:
For the right arm these values are not even initialized. You should test it also with the right arm before merging.